managed_vms/datastore/src/main/java/com/example/managedvms/datastore/DatastoreServlet.java

```diff
@@ -44,17 +46,25 @@
   @Override
   public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException,
       ServletException {
+    // encrypt user ip using PBEWithMD5AndDES
+    BasicTextEncryptor encryptor = new BasicTextEncryptor();
+    encryptor.setPassword(Double.toString(10000*Math.random()));
```

Comment:
Ditto. Not very secure to use Math.random. We should either use a secure random number generator or at least add a "don't do this" disclaimer. http://stackoverflow.com/a/11052736/101923